src/main/java/com/CezaryZal/constants/UserConstants.java

```diff
@@ -0,0 +1,24 @@
+package com.CezaryZal.constants;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserConstants {
```

Comment:
Co to? po co to?

Reply:
Obecnie żebym mógł stworzyć nowe konto muszę najpierw stworzyć nowego użytkownika w aplikacji HealthCalendar, później wraz z jego id stworzyć użytkownika w tej aplikacji.
Żeby stworzyć użytkownika w HC potrzebuje tokenu, a żeby stworzyć token muszę najpierw się uwierzytelnić więc wykorzystuje dane istniejącego użytownika do tego.
Jego dane  przechowuje w zewnętrznym pliku z którego pobieram dane przez tą klasę.
Później będę chciał to zrobić przez pominięcie uwierzytelienia, tylko odrazu stworzenia tokenu.

Reply:
Totalnie nie rozumiem tego konceptu, ale ok. Natomiast kod jest przez to bardzo nieczytelny.